eng/Version.Details.xml

```diff
@@ -1,59 +1,59 @@
+    <Dependency Name="ILLink.Tasks" Version="0.1.6-prerelease.19218.1">
       <Uri>https://github.com/mono/linker.git</Uri>
-      <Sha>1b3d8bc8eb5dd3678dd91eb08157c885336aaea0</Sha>
+      <Sha>c09c490012b1e11cf6d52a6c6a486d4dc5f401bc</Sha>
       <RepoName>linker</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.0.0-preview4-27518-71">
+    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.0.0-preview5-27622-75">
       <Uri>https://github.com/dotnet/coreclr</Uri>
-      <Sha>d833cacabd67150fe3a2405845429a0ba1b72c12</Sha>
+      <Sha>4895a06c0dcc5d13f75dd55bdce83f7792d72ba4</Sha>
     </Dependency>
-    <Dependency Name="NETStandard.Library" Version="2.1.0-prerelease.19158.1">
+    <Dependency Name="NETStandard.Library" Version="2.1.0-prerelease.19222.1">
       <Uri>https://github.com/dotnet/standard</Uri>
-      <Sha>8ef5ada20b5343b0cb9e7fd577341426dab76cd8</Sha>
+      <Sha>5eee83eaa61eff38f470dc690218bebf73f46e23</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="4.6.0-preview4.19163.10">
+    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="4.6.0-preview5.19224.8">
       <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>dc522ef97fac72e64cd74825b7ef497f82af4624</Sha>
+      <Sha>d06ce9d2116cb4bda528822e63d32ce3735ea653</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App" Version="3.0.0-preview4-27518-01">
+    <Dependency Name="Microsoft.NETCore.App" Version="3.0.0-preview5-27626-15">
       <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>ee54d4cbd2b305eadf6f341bdc9d4abccdb50559</Sha>
+      <Sha>61f30f5a23af498908cadd9cc282d4d67d71f522</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="1.0.2-beta5.19115.3">
+    <Dependency Name="Microsoft.DotNet.Common.ItemTemplates" Version="1.0.2-beta5.19223.2">
       <Uri>https://github.com/dotnet/templating</Uri>
-      <Sha>0fd9a865045026041f2ea4942eb8c91782193078</Sha>
+      <Sha>e854a58b469c2b4a660fc6e448657d1dd8e40ab9</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Compilers" Version="3.1.0-beta1-19162-08">
+    <Dependency Name="Microsoft.NETCore.Compilers" Version="3.1.0-beta3-19217-05">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>01f3eb103049e2c93e0516c7d50908031deaca74</Sha>
+      <Sha>79b8271f65609b23e0c0ee1babed6862b7c5ed56</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Build" Version="16.1.0-preview.27">
+    <Dependency Name="Microsoft.Build" Version="16.0.462">
       <Uri>https://github.com/microsoft/msbuild</Uri>
-      <Sha>d004974104fde202e633b3c97e0ece3287aa62f9</Sha>
+      <Sha>62fb89029dac66f73a5ca52363fd11b36f12190b</Sha>
     </Dependency>
-    <Dependency Name="NuGet.Build.Tasks" Version="5.0.0-preview1.5754">
+    <Dependency Name="NuGet.Build.Tasks" Version="5.1.0-rtm.5921">
```

Comment:
How is it that this version number changes but the Sha remains the same?

Reply:
They were not participating in the prodcon build, rebased on your NuGet update.